apps/api/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:16-alpine AS base
+WORKDIR /src/app
+COPY package*.json ./
+
+FROM base AS dependencies
+
+COPY packages/planning-inspectorate-libs packages/planning-inspectorate-libs
+
+COPY ./apps/api .
```

Comment:
Shouldn't we keep the folder structure to make sure npm workspaces are working properly?

So that the CMD will be node `apps/api/...`

Reply:
Can do!